### PR TITLE
Slightly awkward rustup

### PR DESCRIPTION
**Pull request by nrc**
Note that this requires https://github.com/serde-rs/syntex/pull/22 to land before we can pass tests

### CODE REVIEW

File: src/patterns.rs [hunk 1]
```diff
@@ -33,11 +33,8 @@ impl Rewrite for Pat {
...
-            Pat_::PatWild(kind) => {
-                let result = match kind {
-                    PatWildKind::PatWildSingle => "_",
-                    PatWildKind::PatWildMulti => "..",
-                };
+            Pat_::PatWild => {
+                let result = "_";
                 if result.len() <= width {
                     Some(result.to_owned())
                 } else {
```
Comment (johannhof) [inline src/patterns.rs]: lines 36 to 43 can be reduced to `Pat_::PatWild => "_".to_owned()`, no?

File: src/expr.rs [hunk 2]
```diff
@@ -1424,6 +1424,7 @@ fn rewrite_binary_op(context: &RewriteContext,
     // FIXME: format comments between operands and operator
 
     let operator_str = context.snippet(op.span);
+    println!("nrc {:?} {}", op, operator_str);
```
Comment (johannhof) [inline src/expr.rs]: pretty sure this isn't meant to be here :)

File: src/patterns.rs (hunk 1 above)
Comment (nrc) [inline src/patterns.rs]: I don't think so - width might be 0

File: src/expr.rs (hunk 2 above)
Comment (nrc) [inline src/expr.rs]: thanks

**Comment by johannhof** [comment]
I think this should be good to merge

**Pull request merged**